web/gui/dashboard_info.js

```diff
@@ -1314,9 +1330,57 @@ netdataDashboard.context = {
 
     'ip.inerrors': {
         info: 'Errors encountered during the reception of IP packets. ' +
-            '<code>noroutes</code> (<code>InNoRoutes</code>) counts packets that were dropped because there was no route to send them. ' +
-            '<code>truncated</code> (<code>InTruncatedPkts</code>) counts packets which is being discarded because the datagram frame didn\'t carry enough data. ' +
-            '<code>checksum</code> (<code>InCsumErrors</code>) counts packets that were dropped because they had wrong checksum. '
+            '<b>noroutes</b> (<code>InNoRoutes</code>) counts packets that were dropped because there was no route to send them. ' +
+            '<b>truncated</b> (<code>InTruncatedPkts</code>) counts packets which is being discarded because the datagram frame didn\'t carry enough data. ' +
+            '<b>checksum</b> (<code>InCsumErrors</code>) counts packets that were dropped because they had wrong checksum. '
+    },
+
+    'ip.mcast': {
+        info: 'Total multicast traffic in the system.'
+    },
+
+    'ip.mcastpkts': {
+        info: 'Total transferred multicast packets in the system.'
+    },
+
+    'ip.bcast': {
+        info: 'Total broadcast traffic in the system.'
+    },
+
+    'ip.bcastpkts': {
+        info: 'Total transferred broadcast packets in the system.'
+    },
+
+    'ip.ecnpkts': {
+        info: 'Total number of received IP packets with ECN bits set in the system. '+
+        '<b>CEP</b> - congestion encountered. '+
+        '<b>NoECTP</b> - non ECN-capable transport. '+
+        '<b>ECTP0</b> and <b>ECTP1</b> - ECN capable transport.'
+    },
+
+    'ip.tcpreorders': {
+        info: 'TCP prevents out-of-order packets by either sequencing them in the correct order or by requesting the retransmission of out-of-order packets. '+
+        '<b>Timestamp</b> - detected re-ordering using the timestamp option. '+
+        '<b>SACK</b> - detected re-ordering using Selective Acknowledgment algorithm. '+
+        '<b>FACK</b> - detected re-ordering using Forward Acknowledgment algorithm. '+
+        '<b>Reno</b> - detected re-ordering using Fast Retransmit algorithm.'
+    },
+
+    'ip.tcpofo': {
+        info: 'TCP maintains an out-of-order (OOO) queue to keep the OOO packets in the TCP communication. '+
+        '<b>InQueue</b> - the TCP layer receives an OOO packet and has enough memory to queue it. '+
+        '<b>Dropped</b> - the TCP layer receives an OOO packet but does not have enough memory, so drops it. '+
+        '<b>Merged</b> - the received OOO packet has an overlay with the previous packet. '+
+        'The overlay part will be dropped. All these packets will also be counted into <b>InQueue</b>.'
```

Comment:
```suggestion
        'The overlay part will be dropped. All these packets will also be counted into <b>InQueue</b>. '+
        '<b>Pruned</b> - packets dropped from out-of-order queue because of socket buffer overrun.'
```

Reply:
For reference:
https://patchwork.ozlabs.org/project/netdev/patch/1324511401-2640-1-git-send-email-subramanian.vijay@gmail.com/
https://diamond.readthedocs.io/en/latest/collectors/TCPCollector/